### PR TITLE
Using Bootstrap request for review

### PR DESCRIPTION
**Pull request by Joecleverman**
I complete the using bootstrap project (mobile-first for responsive) I would like a TSS to review it, please.

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+linters:
+  stylelint:
+    config: "stylelint.config.js"
+    fixer: true
```
Comment (MauricioRobayo) [inline .stickler.yml]: Use the provided [`.stickler.yml` file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml), you should not make any changes to this file.

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Hi,

Great start, the project is looking good and you are making very nice use of Bootstrap, but need to fix some issues:

* [ ] Fix [HTML validation errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Frawcdn.githack.com%2FJoecleverman%2Fusing-bootstrap%2F5b4bcf5e3104fe8e7264f2548eb3711249421d81%2Findex.html).
* [ ] Enable Stickler on this repository.
* [ ] Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))
* [ ] Verify the path to the images you are using, they are not loading - although this might be a Newsweek issue since I verified on the original page and the images are not loading there either.

#### Footer

* [x] It contains social media icons ([link to page](https://www.newsweek.com/)).

Please update this PR and request a new code review 😊.

Status: __Request changes__.

__[OPTIONAL]__

* [ ] You could of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document.

**Comment by Joecleverman** [comment]
I solve the issues raised by the TSS, but for the HTML validation there is a lot of warnings related to SVG there is nothing I can do about that and some warnings related to h2, h6 I'm using bootstrap there is no need for h2....h6. I do not see what I can do to solve it.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
### Hi @Joecleverman, Nice work.:+1: Here are some changes we need to make  in order to move forward:
1. The body of the page is not centered. We have more whitespace to the right than on the left.

2. The navbar breaks before the medium screen breakpoint. Let's fix this.
image

3. Let us remove the **US** label above the picture in this section:
image

4. Please use appropriately sized images in the **Latest News section**. Consider saving the images directly from that section on the original page.

5. Reduce the margin between image & text in the **More Stories** section.

6. Check the image-text ratio in these columns under the **US, World, Business ...** sections. Use appropriately sized images.
image

7.Let's remove the bg color in the **Culture** section.
image

8. On small screens, the alignment for **Latest News** section is **Image to the left, text to the right,** not text below image.

9. On small screens, remove left margin from social media icons section within the footer.

10. Adjust the font-sizes and style, especially for our article titles to look more like the original.

11. Find workarounds for the markup validation errors. You can add hidden headers to fix the header warnings. They have to be fixed.

_**Kindly make these changes and submit another code review request.**_

**Cheers:smile:**

### Status: Changes requested.

**Comment by Joecleverman** [comment]
All Issue solved. This is a new pull request for a new code review.

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review By Tolulope Olusakin(TSE)
Please check on this in a few minutes for some feedback

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Really great job @Joecleverman :clap: 
Your project is coming together very nicely :+1: Please consider the few comments I've left below

#### Responsiveness
Your project could use a little more responsiveness, please look a little more closely at the sizes of elements, padding of sections, margins, and general element positionings. These are very key as they will affect the look, feel, and more importantly the usability of your sites. I went ahead to attach a few images of areas where I noticed issues with responsiveness, however, please do a full scan of your page just in case I missed or left something out.

The following are some of the responsive issues I've run into:
screenshot
screenshot

Please look into the unnecessary space above/ below sections and take a closer look at your navigation bar

#### Missing social icons
Please take consider the following screenshot, you may find that your social media icons are missing in the footer of your project. Please make the necessary adjustments and request another review!
screenshot

### Summary
There are some sections with too much space above/below them, your footer is missing social media icons and your navigation's styling breaks on mobile. Please attend to these and request another review! All the best :smile: :rocket: 

### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by Joecleverman** [comment]
I'm sorry Oluwadamilareolusakin you just reviewed an old version. I forgot to change the live link. These errors are not in the new version.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @Joecleverman, good work:+1:, but some changes are still required.**

1. The spacing between lines in the title and below the title of the **Featured Stories** section is too much. Let us close up the gap a little.

image



2. Also, in the **More Stories** section, space below the title is too much and text beyond the image height should wrap below the image. Also, remove left margin/padding from the section title.

image

3. **Editor's Picks Section:** reduce spacing between lines in the article title and spacing after article title.

image

4. **On medium screens:** Compare left margin of the Opinion Section and Latest news section with original.

5. **On medium screens, Latest News section:** Article titles should be aligned with the top of the image. Also add some margin below the section title.
image

6. Check the copyright element in the footer section, on medium screens.

image


7.  On small screens, reduce the left and right margins of the sections beginning from **In the magazine** to the section just before the footer. Compare with the original page to see the difference.

8. On small screens, Images in the **More Stories** section appear really small when compared with the original.

image


_**Let us fix these and submit another code review request.**_

**Good luck:stars:**

**Comment by Joecleverman** [comment]
complete all changed requested.

**Review by nikasulo — APPROVED** [review]
Congratulations @Joecleverman! :clap: 
Exceptional work building this and incorporating all the requested changes. You may merge and keep moving forward :+1: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Pull request merged**